drivers/sensor/tsl2540/Kconfig

```diff
@@ -0,0 +1,70 @@
+# TSL2540 Ambient Light Sensor configuration options
+
+# Copyright (c) 2022 T-Mobile USA, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig TSL2540
+	bool "TSL2540 Ambient Light Sensor"
+	default y
+	depends on DT_HAS_AMS_TSL2540_ENABLED
+	select I2C
+	help
+	  Enable driver for TSL2540 sensors.
+
+if TSL2540
+
+config TSL2540_GLASS_ATTENUATION
+	int "Visible light attenuation fixed point (1.00000)"
+	default 227205
+	help
+	  Integer value for a represenation with 5 decimal points.
+	  Typical value is 2.27205 from datasheet.
+	  Example: 1.2 would be 120000
+
```

Comment:
I'm glad you brought it up as  I was debating with myself about the correct implementation.  My thought was that you would only generally only have once type of glass per device.  But, I could see it change if placed on different sides. 

I'll change it to use DTS instead of kconfig.